apis/fluentd/v1alpha1/plugins/custom/custom_types.go

```diff
@@ -0,0 +1,34 @@
+package custom
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/fluent/fluent-operator/apis/fluentd/v1alpha1/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentd/v1alpha1/plugins/params"
+	"strings"
+)
+
+type CustomPlugin struct {
+	Config string `json:"config,omitempty"`
```

Comment:
Can the `config` be null? If not the `omitempty` should be removed.

Reply:
All fixed.